microsoft/botframework-solutions
# [Botskills] Fix list command

## Description
Fixes #1236 

## Testing Steps
1. Open a terminal in `./lib/typescript/botskills` folder.
1. Run `npm install`.
1. Run `npm run build`.
1. Run `npm link` to symlink the package folder.
1. Have a Virtual Assistant (no need for it to be deployed)
1. Open a terminal in the VA root folder
1. Execute the command `botskills list` and check the correct result.
1. Execute the command `botskills list -f <path>` with the path to your `skills.json` file and check the correct result.

## Checklist
